2.4/test/run

```diff
@@ -145,11 +145,34 @@ function test_config_option() {
     docker stop $(get_cid ${name})
 }
 
+function test_text_search() {
```

Comment:
This test is weird to me, it will pass even if we remove the code that uses MONGODB_TEXT_SEARCH_ENABLED. Shouldn't we test for something specific to having that value set to true?